pages/community/styleguide/aboutguide.md

```diff
@@ -0,0 +1,35 @@
+# AsyncAPI Style Guide
+
+## Introduction
+Welcome to the AsyncAPI Style Guide. This page serves as a guideline to contributors on how to write, structure, and format content/documentation.
+
+These guidelines help us in maintaining language, tone, and voice throughout the website. It also makes it easier for multiple contributors to work together on a single doc while ensuring consistency, accuracy, and clarity. 
+
+## Writing Style
+Below are some recommendations to consider when writing your pages.
+
+### Have your target audience in mind
+Before you begin planning and constructing your content, it is important to know who you are writing for and how the readers will benefit from your content.
+This will help you in strategizing and simplifying your writing process.
+
+### Research
+Before creating your first draft, it is beneficial to research similar resources and use them as references. Avoid plagiarism or copying and pasting from other websites unless the content already exists on our site.
+
+### Follow the writing principles
+When writing our content or documentation, we aim to achieve the following:
+- **Clear** - It is very important that your writing is as clear as possible. Avoid using jargon or ambiguous words. 
+Explain any technical terms in simple and easy-to-understand words while considering your target audience. Additionally, keep your sentences short and sweet.
+- **Concise** - When conveying information, it is crucial to be concise. This reduces redundancy and caters to the needs of the reader while providing value.
+- **Consistent** - Maintaining consistency makes it easy for multiple collaborators to work together efficiently. This ensures professionalism while making it easy to implement updates and changes.
+- **Friendly** - Your content/documentation should be relatable. Always use an active voice instead of a passive tone when writing. Moreover, adding some visual graphics and real-life examples helps the reader better understand the information more practically.
```

Comment:
```suggestion
- **Friendly** - Your documentation should be relatable. Always use an active voice instead of a passive tone when writing. Moreover, adding visual graphics and real-life examples helps the reader understand the information more practically.
```